### PR TITLE
Add README.md

### LINKED ISSUE #2: Add Readme.md

### PR DESCRIPTION
**Pull request by brentleyjones**
Closes #2
- [x] Logo
- [x] Build status
  - [x] #12 
- [x] Supported platforms
- [x] Mission statement/purpose
- [x] Features
- [x] Requirements
- [x] Installation
- [x] Contributing
  - [x] #32 
- [x] Code of conduct
- [X] License

### CODE REVIEW

**Comment by brentleyjones** [comment]
The "purpose" is a bit weak, but I think we can strengthen it once we get more of the framework done.

**Comment by brentleyjones** [comment]
I'll do usage later (#31), along with playgrounds.

**Comment by brentleyjones** [comment]
@TabletopAssistant/dicekit This is ready for final review.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
+<img src="Logo/PNG/Header@2x.png" width="128" height="128" title="DiceKit Logo" />
+
+DiceKit
+=======
+
+[![Build Status](https://www.bitrise.io/app/bba3ae57e91a2417.svg?token=xykKCgO1PqORibgpZeeYrw&branch=master)](https://www.bitrise.io/app/bba3ae57e91a2417)
+![Supported platforms](https://img.shields.io/badge/platforms-iOS%20%2B%20OS%20X%20%2B%20watchOS-blue.svg)
```
Comment (brentleyjones) [inline README.md:7]: I'm not so sure about a platforms badge. I think it's probably redundant.

**Comment by LoganJohnson** [comment]
:+1:

File: README.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+[![Latest release](https://img.shields.io/github/release/tabletopassistant/dicekit.svg)](https://github.com/TabletopAssistant/DiceKit/releases)
+[![Carthage compatible](https://img.shields.io/badge/Carthage-compatible-4BC51D.svg)](https://github.com/Carthage/Carthage)
```
Comment (brentleyjones) [inline README.md:9]: So, it's technically not compatible until https://github.com/Carthage/Carthage/issues/340 gets fixed in a release. I think it's probably fine that I leave this here for now?

**Comment by JonathanHoffman** [comment]
The CONTRIBUTING.md link 404's. I assume that will be fixed with #32. Other than that, :+1:

**Pull request merged**